flake8_async/visitors/helpers.py

```diff
@@ -335,12 +348,30 @@ def with_has_call(
             ),
         ):
             assert isinstance(item.item, cst.Call)
-            assert isinstance(res["library"], cst.Name)
+            assert isinstance(res["library"], (cst.Name, cst.Attribute))
             assert isinstance(res["function"], cst.Name)
-            if res["library"].value not in base:
+            library_node = res["library"]
+            for library_str in base:
+                if (
+                    isinstance(library_node, cst.Name)
+                    and library_str == library_node.value
+                ):
+                    break
```

Comment:
yes, though sounds complicated to engineer. I'll give it a try and see if I get anywhere easily, otherwise throw in a `TODO` and maybe deal with it at a later point.

Reply:
Ended up being straightforward to write, but got sidetracked by https://github.com/Instagram/LibCST/issues/1143